samples/k8s-connected-and-managed-cluster/README.md

```diff
@@ -0,0 +1,155 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Connected and Managed Cluster sample
+
+This folder contains samples that show how an ISV can prepare a Kubernetes application that can be installed on Azure Kubernetes Service (AKS) clusters or Azure Arc-enabled Kubernetes Clusters.
+
+Note:
+  - Throughout this sample:
+      - The terms  "connectedClusters" and "connected clusters" denote Azure Arc-enabled Kubernetes clusters.
+      - The terms  "managedClusters" and "managed clusters" denote Azure Kubernetes Service (AKS) clusters.      
+
+## Artifact changes
+
+The following changes are required to enable support for connected and managed clusters.
+
+### Manifest
+
+Add 'supportedClusterTypes' under extensionRegistrationParameters.
+
+Sample below indicates all distributions of connectedClusters type and all ditributions of managedClusters type are supported.
+
+![Alt text](images/Manifest_changes.PNG)
```

Comment:
It seems there are casing error as the image don't show well. Can you fix those? png vs PNG.

Reply:
Fixed the casing mismatches.